### DISCUSSION TITLE
Approval process (Resubmit): Comment section for table entries

### DISCUSSION THREAD

**Discussion by berief** [Budibase/budibase#3941] [Help] [answered]
Is it possible to comment table entries (created via a form) during the approval process and thus send them back suggested changes?

Thank you and best regards

**Comment by mjashanks** [comment, marked as answer]
HI @berief 

The way to do this would be to create a Comments table where `One Item has many Comments`. The guide below shows how you can add comments from the parent item: https://docs.budibase.com/guides/create-a-child-record-from-the-parent

**Comment by berief** [comment]
Thank you @mjashanks
